src/blockchain/log-processors/designated-report-submitted.ts

```diff
@@ -3,7 +3,35 @@ import * as Knex from "knex";
 import { FormattedLog, ErrorCallback } from "../../types";
 
 export function processDesignatedReportSubmittedLog(db: Knex, augur: Augur, trx: Knex.Transaction, log: FormattedLog, callback: ErrorCallback): void {
-    console.log("TODO: DesignatedReportSubmitted");
-    console.log(log);
-    callback(null);
+    // @achapman: "The designated reporter purchases Stake Tokens in order to do the Designated Report"
+    // Should designatedReportSubmitted include StakedToken address?
+
+    // for phase, sqlite lacks an ENUM, but the idea here is that:
+    //  phase:null = no report submitted
+    //  phase:0 = designated report submitted
+    //  phase:1 = round1
+    //  phase:2 = round2
+    // if that seems reasonable, i'll update PR to import the enum and use name below
+    const marketStateDataToInsert: { [index: string]: string | number | boolean } = {
+        marketID: log.market,
+        phase: 0,
+        isDisputed: false,
+        blockNumber: log.blockNumber
+    };
+
+    db.transacting(trx).insert(marketStateDataToInsert).returning("marketStateID").into("market_state").asCallback((err: Error | null, marketStateID: Array<number>): void => {
```

Comment:
Just double-checking b/c there aren't tests for this yet -- is `Array<number>` the right type for `marketStateID` here?  (It's not `Array(<{ marketStateID: number }>)` or something fancy like that?)

Reply:
This is the right type, i have tested this code with the event emitter contract

Reply:
If the callback is passed an Error here, is `marketStateID` undefined / not passed in at all?  If so, the callback's signature should reflect that.